src/mpcCoreKit.ts

```diff
@@ -200,6 +200,39 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     return this.options.uxMode === UX_MODE.REDIRECT;
   }
 
+  // RecoverTssKey only valid for user that enable MFA where user has 2 type shares :
+  // TssShareType.DEVICE and TssShareType.RECOVERY
+  // if the factors key provided is the same type recovery will not works
```

Comment:
we should explicitly throw error in this case or is the duplicate Duplicate TSS Index is the error for this only, in that case plz make error msg more clear.